device-types/Cisco/FPR1K-DT-RACK-MNT.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Cisco
+model: FPR1K-DT-RACK-MNT
+slug: cisco-fpr1k-dt-rack-mnt
+part_number: FPR1K-DT-RACK-MNT
+u_height: 2.0
+is_full_depth: true
+subdevice_role: parent
+airflow: front-to-rear
```

Comment:
I would change this to passive or remove it as there is no active cooling in the RM chassis.